third-party/ios/ExecutorchLib/ExecutorchLib/Utils.hpp

```diff
@@ -12,7 +12,14 @@
 #import <Foundation/Foundation.h>
 #endif
 
+template <typename T> struct DataPtrWithNumel {
```

Comment:
I think you can think of utilizing here `std::span` to avoid raw pointers with their sizes